src/paradigma/gait_analysis_config.py

```diff
@@ -30,10 +30,10 @@ def __init__(self) -> None:
 
         self.time_colname = 'time'
         
-        self.l_accelerometer_cols: List[str] = [
-            DataColumns.ACCELEROMETER_X, 
-            DataColumns.ACCELEROMETER_Y, 
-            DataColumns.ACCELEROMETER_Z
+        self.l_accelerometer_cols: List[str] = [ # strings, not DataColumns, as we'll edit them
```

Comment:
This part is unclear to me, why are we editing col names?

Reply:
I put this under a general config (part of the config refactoring) and set them back to constants. I could not find a place where we changed them.